packages/account/src/Components/forms/personal-details-form.jsx

```diff
@@ -108,11 +108,22 @@ const PersonalDetailsForm = ({
         />
     );
 
+    const poa_clarification_message = localize(
+        'For faster verification, input the same address here as in your proof of address document (see section below)'
+    );
```

Comment:
This is only being used in one place so make it inline and it is best to use the Localize component.

Reply:
Alright , will change it to <Localize/>,
can we keep the variable here in order to not add more lines in render block? 
the same approach is already used in this component for 'name_dob_clarification_message'